### PR TITLE
Added 'CLI for Microsoft 365 v7.1' release announcement

### PR DESCRIPTION
**Pull request by Jwaegebaert**
## Category

- [ ] Content fix
- [x] New article

## Contents of the Pull Request

This PR is the release announcement for the v7.1 release of the CLI for Microsoft 365.

### CODE REVIEW

File: content/cli-for-microsoft-365/cli-for-microsoft-365-v7-1/index.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+githubname: jwaegebaert
+categories:
+  - CLI for Microsoft 365
+images:
+  - images/banner-cli-m365.png
+tags:
+  - CLI for Microsoft 365
+  - Azure Active Directory
+  - SharePoint
+type: regular
```
Comment (martinlingstuyl) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-v7-1/index.md:14]:
```suggestion
type: popular
```

**Review by martinlingstuyl — COMMENTED** [review]
Looks fine @Jwaegebaert! One minor detail that I ran into.

File: content/cli-for-microsoft-365/cli-for-microsoft-365-v7-1/index.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+docker run --rm -it m365pnp/cli-microsoft365:latest
+```
+
+## Need More Information?
+
+For additional guidance on getting started or to explore detailed information about commands, architecture, or the project itself, visit [aka.ms/cli-m365](https://aka.ms/cli-m365).
+
+## Stay Connected!
+
+We value your feedback and are eager to hear from you. If you have any suggestions for improvement or want to engage with our community, you can reach out to us on [GitHub](https://github.com/pnp/cli-microsoft365/issues), [Discord](https://aka.ms/cli-m365/discord), or [X](https://twitter.com/climicrosoft365). Don't hesitate to connect with us. Your input plays a vital role in shaping the future of CLI for Microsoft 365.
```
Comment (milanholemans) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-v7-1/index.md:134]:
```suggestion
We value your feedback and are eager to hear from you. If you have any suggestions for improvement or want to engage with our community, you can reach out to us on [GitHub](https://github.com/pnp/cli-microsoft365/issues), [Discord](https://aka.ms/cli-m365/discord), or [X](https://x.com/climicrosoft365). Don't hesitate to connect with us. Your input plays a vital role in shaping the future of CLI for Microsoft 365.
```

**Review by milanholemans — CHANGES_REQUESTED** [review]
Nice post @Jwaegebaert!
Just one small addition to be future proof 🙂

**Comment by Jwaegebaert** [comment]
Thanks @milanholemans and @martinlingstuyl for the review!

**Pull request merged**